appeals/api/src/database/seed/seed-training.js

```diff
@@ -0,0 +1,24 @@
+import { databaseConnector } from '../../server/utils/database-connector.js';
+import { seedStaticData } from './data-static.js';
+import { seedLPAs } from './seed-lpas.js';
+import { localPlanningDepartmentList } from './LPAs/training.js';
+
+/**
+ * Seed the training database with the required static data
+ *
+ * @throws {Error} If any database operation fails.
+ * @returns {Promise<void>}
+ */
+const seedTraining = async () => {
+	try {
+		await seedStaticData(databaseConnector);
+		await seedLPAs(databaseConnector, localPlanningDepartmentList);
+	} catch (error) {
+		console.error(error);
+		throw error;
+	} finally {
+		await databaseConnector.$disconnect();
+	}
+};
+
+await (process.env.NODE_ENV === 'training' ? seedTraining() : null);
```

Comment:
Not sure we need to set/check `NODE_ENV` (I know copied from the prod one, but I wasn't sure why that was there either!

Reply:
yeah wasn't so sure about that... those are passed to the npm scripts with CROSS-ENV